package.json

```diff
@@ -51,7 +51,6 @@
     }
   },
   "jest": {
-    "testEnvironment": "node",
```

Comment:
Not sure why, but removing it can fix `TypeError: environment.setup is not a function`.

Reply:
Awesome, good finding :) thanks